data/settings/src/main/java/com/google/jetpackcamera/settings/JcaSettingsSerializer.kt

```diff
@@ -35,7 +35,8 @@ object JcaSettingsSerializer : Serializer<JcaSettings> {
         .setStabilizeVideo(VideoStabilization.VIDEO_STABILIZATION_UNDEFINED)
         .setStabilizePreviewSupported(false)
         .setStabilizeVideoSupported(false)
-        .setMaxFrameRate(1)
+        .clearSupportedFrameRates()
```

Comment:
Since this is a `newBuilder()`, calling `clearSupportedFrameRates()` is unnecessary.